gdc_client/upload/client.py

```diff
@@ -431,7 +432,7 @@ def _upload(self):
                     log.error("Can't upload: {}".format(r.content))
                     return
 
-                pbar = tqdm_file(total=self.file_size)
+                pbar = get_file_transfer_pbar(self.file_path, self.file_size, desc="Uploading")
```

Comment:
did the old behavior have the "Uploading" prefix?

Reply:
That `desc` is only used in debug logging, not the progress bars themselves.   So, it's ok.  The additional debug logging is beneficial.